priv/templates/common/error.greenbar

```diff
@@ -1,23 +1,38 @@
-~attachment title="Command Error" color="#ff3333" Caller=$initiator Pipeline=$pipeline_text "Pipeline ID"=$id Started=$started~
-~if cond=$planning_failure ~
-The pipeline failed planning the invocation:
-~br~
+~if cond=$execution_failure != ""~
```

Comment:
Whoa, that's not the easiest template to read. I wonder if it would behoove us to expand on the `if` tag to alleviate some of the duplication. Support for `and`/`or` and maybe `else` would make this a lot easier to read.

Not for this PR to be clear. Just musing about potential improvements.